src/frontends/tensorflow_common/tests/transpose_sinking.cpp

```diff
@@ -1,645 +0,0 @@
-// Copyright (C) 2018-2023 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
```

Comment:
How do you think - is it possible to adopt these tests for the new Transpose Sinking?

Reply:
I think so